funnel/views/participant.py

```diff
@@ -115,20 +115,26 @@ def participant_edit(profile, space, participant):
     return render_form(form=form, title=_(u"Edit Participant"), submit=_(u"Save changes"))
 
 
-@app.route('/<space>/participant', subdomain='<profile>')
+@app.route('/<space>/participant', methods=['GET', 'POST'], subdomain='<profile>')
 @lastuser.requires_login
 @load_models(
     (Profile, {'name': 'profile'}, 'g.profile'),
     ((ProposalSpace, ProposalSpaceRedirect), {'name': 'space', 'profile': 'profile'}, 'space'),
     permission='view')
-def participant(profile, space):
+@requestargs('puk', 'key')
+def participant(profile, space, puk, key):
+    """
+    Endpoint for contact exchange.
+
+    Notice: The GET method to this endpoint is deprecated and will be removed by 1st September, 2018
```

Comment:
Use `TODO` so it can be found later. Or use a Sphinx deprecated tag. We'll forget about this notice.

Reply:
Doing that. I anyway added a reminder for myself at end of august just in case.